### PR TITLE
Add language switching support & welsh content

### PR DESCRIPTION
**Pull request by palequezal**

### CODE REVIEW

File: src/main/play-25/twirl/views/layouts/govuk_template.scala.html [hunk 1]
```diff
@@ -15,23 +15,30 @@
...
 <!DOCTYPE html>
-<!--[if lt IE 9]><html class="no-js lte-ie8" lang="en"><![endif]-->
-<!--[if gt IE 8]><!--><html lang="en" class="no-js"><!--<![endif]-->
+<!--[if lt IE 9]><html class="no-js lte-ie8" lang=@lang><![endif]-->
+<!--[if gt IE 8]><!--><html lang=@lang class="no-js"><!--<![endif]-->
     <head>
         <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
 
-        <title>@title.getOrElse("GOV.UK - The best place to find government services and information")</title>
+        <title>@title.getOrElse(Messages("base.title.default"))</title>
```
Comment (danewatts) [inline src/main/play-25/twirl/views/layouts/govuk_template.scala.html]: You have brought implicit messages in however use the Messages object. Please switch this for all messages.

File: resources/messages.cy [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+base.title.default                              = GOV.UK – Y lle gorau i ddod o hyd i wasanaethau a gwybodaeth y llywodraeth
+base.skip_link                                  = Ewch yn syth i''r prif gynnwys
```
Comment (danewatts) [inline resources/messages.cy]: Replace alll dumb quotes with smart quotes please

File: project/LibDependencies.scala [hunk 3]
```diff
@@ -14,7 +14,9 @@ object LibDependencies {
 
   val test: Seq[ModuleID] = Seq(
     "org.scalatest" %% "scalatest" % "3.0.5" % Test,
-    "org.pegdown"   %  "pegdown"   % "1.6.0" % Test
+    "org.pegdown"   %  "pegdown"   % "1.6.0" % Test,
+    "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % Test,
+    "org.jsoup" % "jsoup" % "1.8.1" % Test
```
Comment (danewatts) [inline project/LibDependencies.scala]: There is more recent compatible versions of the 2 additional libraries added. Can you update them please?

**Review by danewatts — CHANGES_REQUESTED** [review]

**Comment by JackieDev** [comment]
Hi there, I am a dev on Live Services DDCW, we are currently implementing welsh translation on one of our services, we have noticed there is still some english content in the footer - that this PR fixes - that the welsh translation unit could fail us on, does anyone know when this will be merged? Many thanks

**Comment by danewatts** [comment]
@Jackie1988 hey, I have requested changes to this PR. Please address these changes.

**Review by danewatts — APPROVED** [review]

**Comment by Jackie1988** [comment]
Hi @palequezal when will this PR be reviewed and updated? Thanks

**Comment by danewatts** [comment]
@Jackie1988 it is already merged?

**Comment by Jxr227** [comment]
#6 Is this fixed as part of this PR?

**Comment by Jxr227** [comment]
@danewatts Looks like the build failed because it's using oraclejdk8 which looks like it could be this: https://travis-ci.community/t/install-of-oracle-jdk-8-failing/3038

**Comment by danewatts** [comment]
I have reverted this due to a failing build. This is due to cross-compilation for play versions. I am happy to merge this back in once it is tested for both play versions. Please see: https://github.com/hmrc/sbt-play-cross-compilation for reference.

**Pull request merged**